Tests/scripts/test_end_to_end_results.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+echo "Starting the e2e test summary script - Server type: ${SERVER_TYPE}, Product type: ${PRODUCT_TYPE}"
+
+if [[ $CI_COMMIT_BRANCH = "xsoar_8_end_to_end_tests" ]]; then 
```

Comment:
```suggestion
if [ -n "${NIGHTLY}" ]; then
```
revert when done testing